### ISSUE TITLE
ImportError for jinja2.tests

### ISSUE THREAD

**Issue by jrobbins** [GoogleChrome/chromium-dashboard#1204]
**Describe the bug**
When running the app, we get an error:
```  
File "/.../chromium-dashboard/lib/jinja2/defaults.py", line 4, in <module>
    from .tests import TESTS as DEFAULT_TESTS  # noqa: F401
  File "/.../google_appengine/google/appengine/tools/devappserver2/python/runtime/sandbox.py", line 1168, in load_module
    raise ImportError('No module named %s' % fullname)
ImportError: No module named jinja2.tests
```

**To Reproduce**
1. `npm start`

**Additional context**
Possible root causes include
- A name conflict with the tests.py file and our tests directory or something else with the same name.  However, removing our tests directory did not seem to resolve the problem.
- Something changing the python import path?
- Somehow messing up relative imports?

**Work-arounds for now**
When we get off python2, we will redo all of our library dependencies for python3, so this problem should be irrelevant after that.  Until then, you can paste the contents of lib/jinja2/tests.py into the point where it is imported in defaults.py.

**Comment by shivamag00** [comment]
I was investigating this issue. I can confirm that it is being caused due to the name `tests.py`. I renamed `tests.py` to `test.py` (without s) in `lib/jinja2` and the error was gone.

**Comment by yuktibagrodia** [comment]
I tried the same thing, as @shivamag00 said the error is gone by changing the name to test.py

**Comment by jrobbins** [comment]
I can confirm that naming that file 'test.py' works.  Also, renaming it to 'testxs.py' works.

However, renaming it to 'tests2.py' or 'testsx.py' does not work.  I'm confused about that because I didn't think that python module names have any notion of matching a prefix of a module name.   So, maybe something in the app engine dev server is treating modules named "tests.*" specially.

**Comment by jrobbins** [comment]
OK, after a *lot* of poking around looking for things that do something with the string 'tests'.  I found a fairly obvious root cause.

Our skip_files.yaml had a rule to not upload anything named 'tests' to the GAE production environment, and that rule applies to dev_appserver as well.   I simply added a trailing slash to make it exclude directories named tests rather than any file or directory that started with the string 'tests'.
https://github.com/GoogleChrome/chromium-dashboard/commit/8bc41f6598bc510843c68c46c052b6c174a18819
